.github/workflows/ci_cd.yml

```diff
@@ -38,17 +38,31 @@ jobs:
 
       - name: Install dependencies
         run: |
+          sudo apt update
+          sudo apt-get install -y build-essential zip pandoc texlive-latex-extra latexmk texlive-pstricks
           python -m pip install --upgrade pip tox
-      - name: Build documentation
+          python -m pip install -r requirements/requirements_doc.txt
+          python -m pip install --editable .
+      - name: Build HTML documentation
         run: tox -e doc
 
-      - name: Upload Documentation
+      - name: Upload HTML Documentation
         uses: actions/upload-artifact@v2.2.1
         with:
-          name: Documentation
+          name: HTML-Documentation
```

Comment:
Are the dashes necessary here and in line 56? I'd prefer a space but understand that it might not be allowed.

Reply:
These are not necessary. However, from the point of view of filenames, it is better to avoid spaces.